src/lib/idmap/sss_idmap.c

```diff
@@ -426,6 +426,16 @@ enum idmap_error_code sss_idmap_calculate_range(struct sss_idmap_ctx *ctx,
          * explicitly.
          */
         new_slice = *slice_num;
+        min = (rangesize * new_slice) + idmap_lower;
+        max = min + rangesize - 1;
+        for (dom = ctx->idmap_domain_info; dom != NULL; dom = dom->next) {
+                if (check_dom_overlap(&dom->range_params,min, max)) {
+                    /* This range overlaps one already registered
+                     * Fail, because the slice was manually configured
+                     */
+                    return IDMAP_COLLISION;
+                  }
```

Comment:
Sorry for nitpicking here, but the "}" is not indented correctly.

Whoever pushes the patch could fix this as well instead of asking the contributor to re-submit the patch.